_src/assets/js/bundle.min.js

```diff
@@ -1,4 +1 @@
en","false")},n[t].focus=function(){this.querySelector(".tooltip__details").setAttribute("aria-hidden","false")},n[t].onmouseleave=function(){this.querySelector(".tooltip__details").setAttribute("aria-hidden","true")},n[t].blur=function(){this.querySelector(".tooltip__details").setAttribute("aria-hidden","true")},n[t].onkeydown=function(e){return 27==e.which?(this.querySelector(".tooltip__details").setAttribute("aria-hidden","true"),e.preventDefault(),!1):void 0};var r=document.querySelectorAll(".tooltip--bottom");for(t=0;t<r.length;t++){var i=r[t].clientHeight;r[t].style.marginBottom="-"+i+"px"}var o=document.querySelectorAll(".tooltip--top");for(t=0;t<o.length;t++){i=o[t].clientHeight;o[t].style.top="-"+i+"px"}var s=document.querySelectorAll(".tooltip--right");for(t=0;t<s.length;t++){var a=s[t].parentNode.clientWidth;s[t].style.left=a+"px"}var l=document.querySelectorAll(".tooltip--left");for(t=0;t<l.length;t++){a=l[t].clientWidth;l[t].style.left="-"+a+"px"}}(),function(){if(window&&window.addEventListener){function p(){clearTimeout(e),e=setTimeout(t,100)}var t,e,h=Object.create(null),m=function(){},g="http://www.w3.org/1999/xlink";t=function(){function i(){var e;0===--f&&(m=window.MutationObserver?((e=new MutationObserver(p)).observe(document.body,{childList:!0,subtree:!0,attributes:!0}),function(){try{e.disconnect()}catch(e){}}):(document.body.addEventListener("DOMSubtreeModified",p,!1),function(){document.body.removeEventListener("DOMSubtreeModified",p,!1)}))}function e(r){return function(){var e,t=document.body,n=document.createElement("x");r.onload=null,n.innerHTML=r.responseText,(e=n.getElementsByTagName("svg")[0])&&(e.style.position="absolute",e.style.width=0,e.style.height=0,t.insertBefore(e,t.firstChild)),i()}}function t(e){return function(){e.onerror=null,e.ontimeout=null,i()}}var n,r,o,s,a,l,u,c,d,f=0;if(window.XMLHttpRequest&&(a=void 0!==(a=new XMLHttpRequest).withCredentials?XMLHttpRequest:XDomainRequest||void 0),void 0!==a){for(m(),c=document.getElementsByTagName("use"),s=0;s<c.length;s+=1){try{r=c[s].getBoundingClientRect()}catch(e){r=!1}n=(u=c[s].getAttributeNS(g,"href").split("#"))[0],o=u[1],l=r&&0===r.left&&0===r.right,r&&0===r.width&&0===r.height&&!l?n.length&&(!0!==(d=h[n])&&c[s].setAttributeNS(g,"xlink:href","#"+o),void 0===d&&(d=new a,(h[n]=d).onload=e(d),d.onerror=t(d),d.ontimeout=t(d),d.open("GET",n),d.send(),f+=1)):l||void 0!==h[n]||(h[n]=!0)}c="",f+=1,i()}},window.addEventListener("load",function e(){window.removeEventListener("load",e,!1),t()},!1)}}(),console.log("hello main.js"),$(document).ready(function(){stickyHeader(),$(window).resize(function(){stickyHeader()}),$(".jq-modal-trigger").click(function(e){var t=$(this).attr("href");$(t).fadeIn("slow"),e.preventDefault(),hideModal()}),$(".jq-modal").fitVids(),$("ul.ml-tabs li").click(function(){var e=$(this).attr("data-tab");$("ul.ml-tabs li").removeClass("current"),$(".tab-content").removeClass("current"),$(this).addClass("current"),$("#"+e).addClass("current")})});
```

Comment:
Is this a change of the asset, because of change of versions?
Furthermore if this is automatically updated based on the versions inside the package.json.
i would also remove this file from git.